test/cpp/jit/test_misc.h

```diff
@@ -575,15 +575,62 @@ void testTopologicalIndex() {
   }
 }
 
-void invokeTestRecordFunction(at::Tensor& t) {
-  autograd::profiler::GetPackedInputsCallback inputs_cb =
-    [t]() {
-      Stack st;
-      pack(st, t);
-      return st;
-    };
-  autograd::profiler::RecordFunction guard("test", inputs_cb);
-  t.add_(torch::ones_like(t));
+at::Tensor invokeTestRecordFunction(at::Tensor& t) {
+  RECORD_FUNCTION_WITH_INPUTS("test", t);
+
+  auto t2 = t.pow(2);
+  return t2;
+}
+
+static const auto invokeTestRecordFunction_JIT = R"JIT(
+  def forward(t):
+    t2 = t.pow(2)
+    return t2
+)JIT";
+
+at::Tensor invokeTestRecordFunctionJIT(at::Tensor& t) {
+  RECORD_FUNCTION_WITH_INPUTS("test", t);
+
+  auto cu = std::make_shared<script::Module>();
+  script::defineMethodsInModule(
+      cu, invokeTestRecordFunction_JIT, script::nativeResolver, c10::nullopt);
+  auto graph = cu->get_method("forward").graph();
```

Comment:
can't you just do `return cu->get_method("forward").run({t})`?

Reply:
sounds good, more like `return cu->get_method("forward")({t}).toTensor()`